api/routes.py

```diff
@@ -291,6 +291,7 @@ def shared_collection_revoke_loan(collection_name, loan_id):
 
 @app.route("/collections/<collection_name>/loans/<loan_id>/fulfill", defaults=dict(mechanism_id=None))
 @app.route("/collections/<collection_name>/loans/<loan_id>/fulfill/<mechanism_id>")
+@app.route("/collections/<collection_name>/loans/<loan_id>/fulfill/<mechanism_id>")
```

Comment:
I think you accidentally copied this line. Or did you mean to add part to it?

Reply:
I did add part to it, then took it out because it's not necessary for this branch, but I left the copy in place.